src/backend/mscoffobj.c

```diff
@@ -708,7 +725,8 @@ void MsCoffObj::term(const char *objfilename)
         return;
 #endif
 
-    build_syment_table();
+    bool bigobj = scnhdr_cnt > 0xffff;
```

Comment:
Not sure, similar constants are sometimes a bit below. Maybe @etcimon has more experience with the actual limit.